pretty_gpx/common/drawing/drawing_data.py

```diff
@@ -96,3 +99,18 @@ class LineCollectionData(BaseDrawingData):
     def plot(self, ax: Axes, color: str) -> None:
         """Plot the annotation."""
         ax.add_collection(LineCollection(**self.kwargs(), colors=color))
+
+@dataclass
+class PolygonCollectionData:
+    """ShapelyPolygon Data."""
+    polygons: Surface_Polygons
+
+    def plot(self, ax: Axes, color_patch: str, color_background: str) -> None:
+        """Plot the polygons."""
+        ax.add_collection(PatchCollection(self.polygons.exterior_polygons,
+                                          facecolor=color_patch,
+                                          edgecolor=None))
+        if self.polygons.interior_polygons is not None and len(self.polygons.interior_polygons) > 0:
+            ax.add_collection(PatchCollection(self.polygons.interior_polygons,
+                                              facecolor=color_background,
+                                              edgecolor=None))
```

Comment:
Ideally the park/green areas should be connected areas without interior holes no ? The holes we see on the maps are building lying on the park but it's still in the park

It seems like we don't really need to manage the case of interior holes and could simply convert them to `PolyFillData` that can be filled entirely

![image](https://github.com/user-attachments/assets/2bd32438-a3ea-4a5a-820e-d72eb72f6c74)

Reply:
![Update_rivers_forests](https://github.com/user-attachments/assets/11b35d44-3f4c-422a-bfc9-ccca22e1e71c)
I just tested by removing the polygon interior list (see commit). I get a much better result